### PR TITLE
Index

### PR DESCRIPTION
**Pull request by edewit**
changed the style of the index page to match the rest + updated the text a bit to make more clear what the links are. @rhoads-zach could you review? It might need some wording / text updates

### CODE REVIEW

**Comment by edewit** [comment]
part of https://github.com/fabric8-launcher/launcher-planning/issues/86

File: docs/landing-page/master.adoc [hunk 1]
```diff
@@ -2,95 +2,178 @@ include::topics/templates/document-attributes.adoc[]
...
-<section _ngcontent-c3="" id="runtimes">
-  <div _ngcontent-c3="" class="container">
-    <div _ngcontent-c3="" class="row">
-      <div _ngcontent-c3="" class="col-md-12">
-        <h1 _ngcontent-c3="">Supported Runtimes</h1>
+<section id="runtimes">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <h3>Documentation on the Supported Runtimes</h3>
```
Comment (tradej) [inline docs/landing-page/master.adoc]: Please, replace _on_ with _about_.

Comment (quintesse) [reply docs/landing-page/master.adoc]: "on" is correct here, isn't it?

Comment (tradej) [reply docs/landing-page/master.adoc]: The IBM Style Guide, which is the authoritative guide on writing documentation, mandates we do not use it in this context.

File: docs/landing-page/master.adoc [hunk 2]
```diff
@@ -2,95 +2,178 @@ include::topics/templates/document-attributes.adoc[]
...
-        <div _ngcontent-c3="" class="fa icon swarm"></div>
-        <p _ngcontent-c3="">
-          WildFly Swarm offers an innovative approach to packaging and running 
-          Java EE applications by packaging them with just enough of the server 
-          runtime to "java -jar" your application.
+    <div class="row">
+      <div class="col-sm-3 runtime">
+        <div class="fa icon swarm"></div>
+        <p>
+          Learn how to use launcher for Wildfly swarm projects
```
Comment (tradej) [inline docs/landing-page/master.adoc]: Please, replace _Wildfly swarm_ with _WildFly Swarm._

File: docs/landing-page/master.adoc [hunk 3]
```diff
@@ -2,95 +2,178 @@ include::topics/templates/document-attributes.adoc[]
...
 
-      <div _ngcontent-c3="" class="col-sm-3 runtime">
-        <div _ngcontent-c3="" class="fa icon boot"></div>
-        <p _ngcontent-c3="">
-          Spring Boot makes it easy to create stand-alone,
-          production-grade Spring based Applications that you can "just run".
+      <div class="col-sm-3 runtime">
+        <div class="fa icon boot"></div>
+        <p >
+          Learn how to use launcher for Spring boot projects
```
Comment (tradej) [inline docs/landing-page/master.adoc]: Please, replace _Spring boot_ with _Spring Boot._

**Comment by edewit** [comment]
@rhoads-zach @tradej so it's okay like this?

**Comment by tradej** [comment]
No complaints on my side, but I am not sure how the situation with separating Node.js visually panned out, so let's wait for @rhoads-zach on this one pls.

**Comment by rhoads-zach** [comment]
@tradej @edewit will merge for now and iterate.

**Comment by rhoads-zach** [comment]
merged: bc227bef92ad69dce8594bf215076808a054a533

**Pull request closed without merging**